README.md

```diff
@@ -251,6 +251,10 @@ environment variable:
   can recognize false positives by "<untagged>" occurring in the message -- this
   indicates a pointer that was cast from an integer, so Miri was unable to track
   this pointer.
+* `-Zmiri-compare-exchange-weak-failure-rate=<rate>` changes the failure rate of
```

Comment:
Please keep the list alphabetically sorted.

Reply:
done